draft-wang-ppm-differential-privacy.md

```diff
@@ -427,13 +427,13 @@ Differential privacy noise sometimes needs to be calibrated based on the
 `SENSITIVITY` of the aggregation function used to compute the aggregate result
 over Client measurements. Sensitivity characterizes how much a change in a
 Client measurement can affect the aggregate result. In this document, we focus
-on the L1 and L2 sensitivity. We define them as a function over two neighboring
-Client measurements:
+on the L1 and L2 sensitivity. We define them as a function over two aggregate
+results from neighboring datasets:
 
 * L1 Sensitivity: the sum of the absolute values of differences at all coordinates
-  of the neighboring Client measurements.
+  of the neighboring aggregate results.
 * L2 Sensitivity: the sum of the squares of the differences at all coordinates of
```

Comment:
While you are at it, can you help change line 435 to be "**the square root** of the sum of squares..."? Thank you.